projects/RPi/options

```diff
@@ -17,7 +17,12 @@
         # arm1136j-s arm1136jf-s mpcore mpcorenovfp arm1156t2-s
         # arm1176jz-s arm1176jzf-s cortex-a8 cortex-a9 cortex-r4
         # cortex-r4f cortex-m3 cortex-m1 xscale iwmmxt iwmmxt2 ep9312.
-        TARGET_CPU="arm1176jzf-s"
+
+        if [ "$DEVICE" = "RPi" ]; then
+          TARGET_CPU="arm1176jzf-s"
+        elif [ "$DEVICE" = "RPi2" ]; then
+          TARGET_CPU="cortex-a7"
+        fi
```

Comment:
I actually considered this, however then you have more dangling files to manage.